gst/gstvalue.h

```diff
@@ -26,6 +26,17 @@
 
 G_BEGIN_DECLS
 
+/**
+ * GST_PRINTABLE_CHAR_ARGS:
+ * @c: a char value to output
+ *
+ * To be used with %c when chars may be invalid. Ensures that c is printable by
+ * substituting invalid chars by '?'. This avoids producing invalid strings,
+ * glib warnings and assertions.
+ */
+#define GST_PRINTABLE_CHAR_ARGS(c) \
+        ((c) < ' ' || (c) & 0x80 ? ((gchar)'?') : ((gchar)(c)))
```

Comment:
this would be a perfect example where using G_UNLIKELY is recommended

Reply:
Done. BTW, G_LIKELY and G_UNLIKELY are not worth its use in general, it has been demonstrated that it does not improve performance in most cases, and can actually degrade it. Branch prediction in modern processors made it obsolete. Does no harm other than decreased readability for no purpose. Only in very critical code it may worth it's use, and usually requires benchmarking to ensure that has no negative effect in performance.